xbmc/settings/AdvancedSettings.h

```diff
@@ -287,6 +287,8 @@ class CAdvancedSettings : public ISettingCallback, public ISettingsHandler
     int m_iEdlMaxStartGap;          // seconds
     int m_iEdlCommBreakAutowait;    // seconds
     int m_iEdlCommBreakAutowind;    // seconds
+    bool m_bEdlCommAutoSkip; /*!< @brief true to automatically skip commercial breaks in Edit Decision Lists by defeault */
```

Comment:
this is dead code. the average user is not aware of advanced settings. please drop.